internal/provisioner/fluentbit.go

```diff
@@ -33,24 +35,41 @@ func newFluentbitHandle(provisioner *KopsProvisioner, awsClient aws.AWS, kops *k
 		return nil, errors.New("cannot create a connection to Fluentbit if the Kops command provided is nil")
 	}
 
+	version, err := cluster.GetUtilityVersion("fluentbit")
```

Comment:
Does it make sense to just pass in the version for the utility instead of the whole cluster object? 0/5 on this.

Reply:
yes it does, for Prometheus I pass `cluster` for other reasons and it looks like I just dumbly copied that pattern